control-plane/cmd/webhook-kafka/main.go

```diff
@@ -131,9 +128,6 @@ func NewValidationAdmissionController(ctx context.Context, _ configmap.Watcher)
 
 		// Whether to disallow unknown fields.
 		true,
-
-		// Extra validating callbacks to be applied to resources.
-		validationCallbacks,
```

Comment:
why remove?

Reply:
Because it was added before thinking that adding this callback here would automatically make the webhook validate the broker resource. It doesn't work that way.

To make that callback getting triggered, we need to register the resource. And when we do register the resource, we don't need this callback as we can use the proper validation in `BrokerStub`.

Use case for this callback is something different: being able to validate an object that's not implementing `apis.Validatable`, such as a Pod. Callback treats everything as `unstructured` so, that's how it works.